multi_dagrun/operators/multi_dagrun.py

```diff
@@ -1,37 +1,32 @@
-from datetime import datetime
-
 from airflow import settings
 from airflow.models import DagBag
 from airflow.operators.dagrun_operator import DagRunOrder, TriggerDagRunOperator
 from airflow.utils.decorators import apply_defaults
 from airflow.utils.state import State
+from airflow.utils import timezone
 
 
 class TriggerMultiDagRunOperator(TriggerDagRunOperator):
     CREATED_DAGRUN_KEY = 'created_dagrun_key'
 
     @apply_defaults
     def __init__(self, op_args=None, op_kwargs=None,
-                 provide_context=None, *args, **kwargs):
+                 *args, **kwargs):
         super(TriggerMultiDagRunOperator, self).__init__(*args, **kwargs)
         self.op_args = op_args or []
         self.op_kwargs = op_kwargs or {}
-        self.provide_context = provide_context
 
     def execute(self, context):
-        if self.provide_context:
-            context.update(self.op_kwargs)
-            self.op_kwargs = context
-
+        context.update(self.op_kwargs)
```

Comment:
It's a not good idea to made updates to a mutable incoming argument.

Reply:
Noted. Switched back to the original formulation